src/main/java/org/cactoos/experimental/Threads.java

```diff
@@ -29,6 +29,10 @@
  * @param <X> The type of result objects.
  *
  * @since 1.0.0
+ * @todo #972:30min Add a new implementation which receive the tasks and able:
+ *  - create a new thread pool with predefined/specified size
+ *  - execute tasks concurrently
+ *  - shutdown the thread pool
  */
 public interface Threads<X> extends Iterable<X> {
```

Comment:
@dgroup I'm now wondering why this isn't an `Iterable<Future<X>>`? Also, let's use `T` instead of `X` in order to follow the JDK standard

Reply:
Basically, in this class, we also fetching the values from the future. Hmm, looks like it has several tasks for single class, so it should be separated. Ok, agree.